### PR TITLE
bugfix/385 (external)

### LINKED ISSUE #385: SBGL coordinate letter issue
SBGL fails to load due to coordinate letter issue.

~~(Please don't yet bother to try to fix as I'll try to create a pull req for this in a moment).~~ Pull req created.

### PR DESCRIPTION
**Pull request by ij1**
Fix SBGL coordinate letter issue.

### CODE REVIEW

**Comment by indianbhaji** [comment]
@ij1 That's great! Could you also add an entry in the document `CHANGELOG.md`?

**Comment by ij1** [comment]
Added one now although I'm somewhat unsure whether the placement of it is correct or not.

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 ### Bugfixes
 - Adds additional check for `undefined` in `CommandParser` when adding args to a `CommandModel` [#364](https://github.com/openscope/openscope/issues/364)
+- Fixes coordinate letter issue at SBGL [#385](https://github.com/openscope/openscope/issues/385)
```
Comment (n8rzz) [inline CHANGELOG.md]: this is exactly what we're looking for in a changelog entry.  good work! 👍

**Comment by erikquinn** [comment]
@ij1 Thanks for this! In order for our tests to run on this, I had to move it to a branch within the openscope repository, since it currently does not work from forks.

I have added you to the `Simulator Collaborators` team on the openScope organization, so you will have access to that branch (`bugfix/385`) to make changes as you wish following any reviews. And if you haven't already, feel free to join us on [the openscope slack](http://openscopeatc.slack.com) (click [here](http://slack.openscope.co) for an invite if you need one)!

Closing this PR in favor of #389.

**Pull request closed without merging**